extensions/mssql/package.json

```diff
@@ -949,7 +931,29 @@
       ]
     }
   },
+  "dependencies": {
+    "buffer-stream-reader": "^0.1.1",
+    "bytes": "^3.1.0",
+    "dataprotocol-client": "github:Microsoft/sqlops-dataprotocolclient#1.0.0",
+    "error-ex": "^1.3.2",
+    "figures": "^2.0.0",
+    "find-remove": "1.2.1",
+    "fs-extra": "^3.0.1",
+    "kerberos": "^1.1.2",
+    "request": "^2.88.0",
+    "request-promise": "^4.2.2",
+    "service-downloader": "github:anthonydresser/service-downloader#0.1.6",
+    "stream-meter": "^1.0.4",
+    "through2": "^3.0.1",
```

Comment:
Added through-2 and tought-cookie in this change. Both were already in yarn.lock via transitive dependency chaining (e.g. we already had these, I just made it official here)